README.md

```diff
@@ -8,10 +8,10 @@
 
 ## Project directory
 
-- Undestanding
+- Understanding
   - [Specification](https://github.com/solid/solid-spec)
   - [Talks](https://github.com/solid/talks)
-  - [Tutorials](https://bit.lu/solid-tutorial)
+  - [Tutorials](https://melvincarvalho.gitbooks.io/solid-tutorials/content/solid_spec.html)
```

Comment:
I think this is a better link: https://melvincarvalho.gitbooks.io/solid-tutorials/content/index.html

Do you want to change it yourself, so I can merge your PR? :)